src/styles/main.scss

```diff
@@ -1,3 +1,53 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+$days: ("mon":0, "tue":1, "wed":2, "thu":3, "fri":4, "sat":5, "sun":6);
+$cell_width:100px;
+$gap:1px;
+$month_length:(28,29,30,31);
+@mixin calendar_block_cell {
+  width: 100px;
+  height: 100px;
+  background-color: #eee;
+  border: 1px solid black;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  max-width: 800px;
+  padding: 10px;
+  @for $i from 1 through 31 {
```

Comment:
Please consider adding an empty line to improve the readability of your code. This small adjustment can make a big difference in how easily others can understand and work with your code.